index.rst

```diff
@@ -1,29 +1,57 @@
-:tocdepth: 1
+:tocdepth: 2
 
 .. sectnum::
 
 .. Metadata such as the title, authors, and description are set in metadata.yaml
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
+.. warning::
```

Comment:
Remove this :)

Reply:
Removed the extraneous comments. Modified note to read "This technote is being reviewed before official release."